proto/tendermint/abci/types.proto

```diff
@@ -262,6 +301,34 @@ message ResponseApplySnapshotChunk {
   }
 }
 
+message ResponsePrepareProposal {
+  bool                             modified_tx             = 1;
+  repeated TransactionRecord       txs                     = 2;
+  bytes                            app_hash                = 3;
+  repeated ExecTxResult            tx_results              = 4;
+  repeated ValidatorUpdate         validator_updates       = 5;
+  tendermint.types.ConsensusParams consensus_param_updates = 6;
+  reserved 7; // Placeholder for app_signed_updates in v0.37
```

Comment:
I don't think it makes sense to add this because:
a) we don't have a similar reserved field for giving the app the data in `RequestPrepareProposal`
b) I don't think we're 100% sure yet that we need to have `app_signed_updates` returned (and in a block)

Reply:
Regarding a), the data is coming in `LastCommitInfo` (see changes to VoteInfo below in lines 415-416)
As for b), when can always "remove" a reserved field later if it turns out not to be necessary/wanted.

Having said this, I'm open to removing this line completely. But then we'd need to be consistent and also remove lines 415-416.

Reply:
I guess we can leave it for now - I just don't think we're that sure of it nor that there's a strong need to have future fields in the proto files

Reply:
I think it's fine to leave it, but it's also not really necessary since it's not as if there is a gap in the numbering that we need to protect from being filled in. YMMV.

The main value of it IMO is the documentation. 🙂

Reply:
OK. Let's leave it for the moment then